code/src/java/pcgen/pluginmgr/PluginManager.java

```diff
@@ -80,17 +82,19 @@ public int compare(InteractivePlugin arg0, InteractivePlugin arg1)
 	public void startAllPlugins()
 	{
 		PCGenMessageHandler dispatcher = msgHandlerMgr.getPostbox();
-		for(InteractivePlugin plugin : pluginMap.keySet())
+		for(final Entry<InteractivePlugin, Boolean> interactivePluginBooleanEntry : pluginMap
```

Comment:
Why such a long localized/temporary variable name?  "entry" would be sufficient, IMHO.  Also, not sure it needs to be "final" - another case where I don't think it adds much value and just adds another copy of a keyword for people to have to analyze